templates/index.html

```diff
@@ -22,7 +22,7 @@
               <svg class="bd-placeholder-img card-img-top" width="100%" height="225" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="{{ dataset["title"] }}" preserveAspectRatio="xMidYMid slice" focusable="false"><title>{{ dataset["title"] }}</title><rect width="100%" height="100%" fill="#55595c"/><text x="50%" y="50%" fill="#eceeef" dy=".3em">{{ dataset["title"] }}</text></svg>
             {% endif -%}
             <div class="card-body">
-              <p class="card-text">{{ dataset["description"]|safe }}</p>
+              <p class="card-text">{{ dataset["home_description"]|safe }}</p>
```

Comment:
```suggestion
              <p class="card-text">{{ dataset["short_description"]|safe }}</p>
```

Talking with @nbodony about this, we already have this and should just use this.